### PR TITLE
BC-3975 - cleanup board code

### PR DESCRIPTION
**Pull request by hoeppner-dataport**
# Short Description
Cleanup the code for the board regarding e.g. function names, event names and events (merge similar events) in general.

## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-3975

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/plugins/application-error-handler.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { applicationErrorModule } from "@/store";
 import { ApplicationError } from "@/store/types/application-error";
 
 export const handleApplicationError = (err: Error) => {
-	console.error(err);
 	/**
 	 * Note: The Global-ErrorHandler wraps the error
 	 * so we can't use instanceof ApplicationError here.
 	 */
+	console.error(err);
```
Comment (NFriedo) [inline src/plugins/application-error-handler.ts:9]: Is there a reason why we added the console.error again or did we forgot to remove it?

File: src/components/feature-board/board/BoardColumnHeader.unit.ts
Comment (NFriedo) [inline src/components/feature-board/board/BoardColumnHeader.unit.ts:1]: When I run the tests in this file I got some console errors/ Vue warnings, eg. "Missing required prop", "Injection "i18" not found". 
Maybe we should have a look and try to fix them.

File: src/components/feature-board/board/Board.unit.ts
Comment (NFriedo) [inline src/components/feature-board/board/Board.unit.ts:1]: I was wondering if we should also add a test for the board title or is it covered by another test?

File: src/components/feature-board/board/Board.vue [hunk 2]
```diff
@@ -43,87 +46,106 @@ import { useRoute } from "vue-router/composables";
...
+			if (horizontalCursorKeys.includes(keyString)) {
+				const change = keyString === "ArrowLeft" ? -1 : +1;
+				columnMove.addedIndex = columnIndex + change;
+				await moveColumn(columnMove);
+			}
 		};
 
-		const onAddColumnWithCard = (cardId?: string) => {
-			addNewColumn(cardId);
+		const onUpdateCardPosition = async (_: unknown, payload: CardMove) => {
```
Comment (NFriedo) [inline src/components/feature-board/board/Board.vue:128]: Do we need the typing here? Or can we just write
`(_, payload: CardMove` ?

Comment (hoeppner-dataport) [reply src/components/feature-board/board/Board.vue:128]: unsure - but let's keep it, as it is not wrong.

**Review by NFriedo — APPROVED** [review]

**Pull request merged**